videoInputSrcAndDemos/libs/videoInput/videoInput.cpp

```diff
@@ -1770,6 +1763,7 @@ static bool setSizeAndSubtype(videoDevice * VD, int attemptWidth, int attemptHei
 	VIDEOINFOHEADER *pVih =  reinterpret_cast<VIDEOINFOHEADER*>(VD->pAmMediaType->pbFormat);
 
 	//store current size
+	// XX: Should they be reset somewhere?
```

Comment:
the tmpType restores all properties including size back to what they were if its not successful. 
so should be good.

Reply:
The thing that confused me is that tmpWidth and tmpHeight aren't used anywhere (and I get an according warning about them), so if they are not needed to restore the state, then those lines can probably be removed entirely.

Reply:
oh yeah - sorry can def be removed. I think that was from some older code when I was doing things differently.

Reply:
As this PR is already merged, I'll leave the removal to you if that is ok.